sycl/doc/design/KernelProgramCache.md

```diff
@@ -215,7 +215,7 @@ kernel of this program will be cached also.
 All requests to build a program or to create a kernel - whether they originate
 from explicit user API calls or from internal SYCL runtime execution logic - end
 up with calling the function
-[`getOrBuild()`](https://github.com/intel/llvm/blob/sycl/sycl/source/detail/program_manager/program_manager.cpp#L149)
+[`getOrBuild()`](https://github.com/intel/llvm/blob/sycl/sycl/source/detail/program_manager/program_manager.cpp#L144)
```

Comment:
I think I would prefer that we remove the link altogether rather than try to keep up with source-line movement.

Reply:
That was my initial thought too, since it's a moving target we link to. Only downside is that it was very useful when reading the documentation.

Reply:
Makes sense, that's a good compromise. I've updated the patch.